src/main/java/seedu/address/model/person/client/Client.java

```diff
@@ -0,0 +1,73 @@
+package seedu.address.model.person.client;
+
+import java.util.Set;
+
+import seedu.address.model.person.Address;
+import seedu.address.model.person.Email;
+import seedu.address.model.person.Name;
+import seedu.address.model.person.Person;
+import seedu.address.model.person.Phone;
+import seedu.address.model.tag.Tag;
+
+/**
+ * Represents a Client in the address book.
+ * Guarantees: name is present and not null, field value is validated, immutable.
+ */
+public class Client extends Person {
+
+    /**
+     * Constructs a client with name given by user and the rest of the fields set to default values.
+     * @param name String representing name of the client
+     */
+    public Client(String name) {
+        super(new Name(name), new Phone("000"), new Email("notSpecified@gmail.com"), new Address("notSpecified"),
+                Set.of(new Tag("notSpecified")));
```

Comment:
Tag might be redundant here, since we are abstracting the `Client` tag into a new class. Perhaps we can look at removing the tag since it is irrelevant to our project, unless we have other uses for it?

Reply:
Another parameter that might be useful is a `Collection` of `Project` objects, which can be displayed in the UI later on.

Reply:
> Tag might be redundant here, since we are abstracting the `Client` tag into a new class. Perhaps we can look at removing the tag since it is irrelevant to our project, unless we have other uses for it?

I see why it might be so...My idea is that the tag in Client class would be something along the lines of specifying client priority, role (e.g. Manager, Team Lead, etc.) and possibly anything else that helps us differentiate clients later on. Does that seem like a good implementation?

Reply:
> Another parameter that might be useful is a `Collection` of `Project` objects, which can be displayed in the UI later on.

Noted and Added!

Reply:
> > Tag might be redundant here, since we are abstracting the `Client` tag into a new class. Perhaps we can look at removing the tag since it is irrelevant to our project, unless we have other uses for it?
> 
> I see why it might be so...My idea is that the tag in Client class would be something along the lines of specifying client priority, role (e.g. Manager, Team Lead, etc.) and possibly anything else that helps us differentiate clients later on. Does that seem like a good implementation?

I would suggest not including the `Tag` for now, since it is not in our current intended feature list. It may be better to add `Tag` later on if we are intending to use it but it would be better not to have it now to reduce the lines of "dead code" in our project.

Reply:
Noted! Will make the changes in the next commit.